_includes/tutorials/kafka-connect-datagen-ccloud/kafka/markup/dev/setup-ccloud.adoc

```diff
@@ -0,0 +1,5 @@
+If this is your first time using link:https://confluent.cloud[Confluent Cloud], follow the manual steps in the link:https://docs.confluent.io/current/quickstart/cloud-quickstart/index.html[Confluent Cloud Quick Start].
+Be sure to also install the link:https://docs.confluent.io/current/cloud/cli/install.html[Confluent Cloud CLI].
+Use the promo code `CC100KTS` to receive an additional $100 free usage (https://www.confluent.io/confluent-cloud-promo-disclaimer[details]).
+
+If you are already a Confluent Cloud user, consider running the link:https://github.com/confluentinc/examples/tree/latest/ccloud/ccloud-stack[ccloud-stack utility] to create a new stack in Confluent Cloud for this tutorial.
```

Comment:
For someone totally new to our demos, the link takes them to the examples repo subpage for ccloud-stack. That page lacks some initial direction for "clone examples, cd to ccloud/ccloud-stack".  IMO somone landing there very new might get stuck.  Just making an observation that the page isn't toally independent and maybe we should add those additional steps there (or here).  WDYT?

Reply:
@rspurgeon very good question that we haven't discussed yet but worthy to sync on, especially if there will be other CCloud tutorials.

There are a few options:

1. Point users to `ccloud-stack` and that's it (as done here). If the README needs steps like clone/cd, I can add that in.
2. Remove mention of `ccloud-stack` completely
3. Pull in all required steps to run `ccloud-stack`.   But couldn't the same argument be made when telling users to follow the Confluent Cloud Quick start (it's a link to docs).
4. Other?

Reply:
For students wishing to do things point and click manually, the quick start seems good IMO.

One possibility: For others wanting something automated, we could suggest an automated approach (in additon to the quickstart way) by integrating `ccloud-stack` either by duplication or introducing a script that does it for the student as part of the ccloud tutorials.    Something like (pseudo): `wget http://gitdata/confluentinc/examples/latest/ccloud/ccloud_stack.sh` and then use the script from there and have that be in a script the student runs.

Reply:
Yeah, the description you wrote is what I meant by option 3 above. The advantages of going this route include:

1. Helps users with more guidance on how to run `ccloud-stack`
2. Eliminates problems with user errors related to wrong credentials, no ACLs for service accounts, etc (the who class of user errors we encountered in our other demos due to environment not being set properly)
3. Potentially.....more easily allows for future automation to validate the tutorial

@rspurgeon so if this is your leaning as well, I'll implement (wget library, run code).  WDYT?

Reply:
I think it would be a nice addition, but maybe we should confer w/ the broader KT team to ensure they are ok w/ bringing that into the fold?  Your call